src/taskEvents/roleElement.js

```diff
@@ -0,0 +1,30 @@
+import { createElement, removeSkillFromUser } from './utils.js';
+import { modalOverlay } from './script.js';
+
+function roleElement(roleTagname, roleLevel, roleTagnameid, userid) {
```

Comment:
```suggestion
function roleElement(roleTagName, roleLevel, roleTagNameId, userId) {
```
Please try to follow the camelCase naming convention

Reply:
`tagname` is the complete key in backend items object i think we can avoid confusion if we don't change the case in between,used camelCase at id for last two params

Reply:
> `tagname` is the complete key in backend items object i think we can avoid confusion if we don't change the case in between,used camelCase at id for last two params

Please share the link to the code having `tagname` as key in our backend if possible

Reply:
Please change the naming for all variables across your changes like `userid` [here](https://github.com/Muralidhar22/website-dashboard/blob/feat/task-leveling-api/src/taskEvents/script.js#L101), `Allroles` [here](https://github.com/Muralidhar22/website-dashboard/blob/feat/task-leveling-api/src/taskEvents/script.js#L102), `userid` [here](https://github.com/Real-Dev-Squad/website-dashboard/blob/ace62557dfa297c2107e1dd08a9aaeb06f2ed01d/src/taskEvents/utils.js#L183) and all other variables.

I would suggest adding [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) extension(if using V S Code) which helps catch common spelling errors and incorrect naming conventions

Reply:
thankyou, now it's changed in the backend ref: https://github.com/Real-Dev-Squad/website-backend/pull/815#discussion_r1035064762 , will make the change here as well

Reply:
done